pootle/apps/pootle_store/unit/timeline.py

```diff
@@ -219,6 +251,29 @@ def submissions(self):
                                        creation_time=self.object.commented_on)
                               .order_by("id"))
 
+    @cached_property
```

Comment:
thinking about it you are right, i suppose i just wonder if we should turn it into a list